src/prisma/schema.prisma

```diff
@@ -140,3 +140,9 @@ model Comment {
   @@index([authorId])
   @@index([postId])
 }
+
+model wcwTopic {
+  id    Int     @id @default(autoincrement())
+  topic String  @db.Text
+  used  Boolean
+}
```

Comment:
Add timestamp to this model. I'm just realizing that a lot of the other models don't have `createdAt` and `updatedAt` so make sure all models have these two fields.

```suggestion
  createdAt DateTime      @default(now())
  updatedAt DateTime      @updatedAt
}
```